pkg/cloudprovider/aws/aws.go

```diff
@@ -22,18 +22,26 @@ import (
 	"net"
 	"regexp"
 	"strings"
+	"sync"
 
 	"code.google.com/p/gcfg"
 	"github.com/mitchellh/goamz/aws"
 	"github.com/mitchellh/goamz/ec2"
+	"github.com/mitchellh/goamz/elb"
```

Comment:
This isn't currently imported into our repo:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/Godeps/Godeps.json#L329